superset-frontend/src/chart/Chart.jsx

```diff
@@ -157,7 +159,7 @@ class Chart extends React.PureComponent {
       // Load saved chart with a GET request
       this.props.actions.getSavedChart(
         this.props.formData,
-        this.props.force,
+        this.props.force || getUrlParam(URL_PARAMS.force), // allow override via url params force=true
```

Comment:
what does URL_PARAMS.force do?

Reply:
URL_PARAMS is an enum that represents the key for url param we want to pull. In this case it is `force`